ext/drreg/drreg.c

```diff
@@ -397,30 +397,19 @@ drreg_event_bb_insert_early(void *drcontext, void *tag, instrlist_t *bb, instr_t
     return DR_EMIT_DEFAULT;
 }
 
-static dr_emit_flags_t
-drreg_event_bb_insert_late(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst,
-                           bool for_trace, bool translating, void *user_data)
+static drreg_status_t
+drreg_insert_restore(void *drcontext, instrlist_t *bb, instr_t *inst, bool force_restore,
```

Comment:
Suggest: tweak the name to make it clearer it restores *all*, to distinguish from other restore-single-reg routines in this file.  Maybe just add _all to the end?

Reply:
Done